### DISCUSSION TITLE
findBy on ManyToMany relation

### DISCUSSION THREAD

**Discussion by wahu0001** [doctrine/doctrine2#10339] [Support Questions]
With my ManyToMany relation between User and Area is it possible to make a findBy on one of this entity like that  

`$user = $this->userRepository->find(1);

$this->areaRepository->findBy(['users'=>$user]);`

and then fetch all areas associated to $user

**Comment by chidicoder** [comment]
I do not think that this is possible, due to the fact that the fields on Course is declared as $user (due to the ManyToMany relationship), and not $user.
